### PR TITLE
#662 created test for release binaries

### PR DESCRIPTION
**Pull request by krzyk**

### CODE REVIEW

**Comment by alex-palevsky** [comment]
Thanks for your pull request, let me find someone who can review it

**Comment by alex-palevsky** [comment]
@longtimeago help us to review this

File: src/test/java/com/rultor/agents/github/ReleaseBinariesTest.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+    /**
+     * ReleaseBinaries should attach artifact to release.
+     * @throws Exception In case of error
+     */
+    @Test
+    @Ignore
+    public void attachesBinaryToRelease() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
```
Comment (longtimeago) [inline src/test/java/com/rultor/agents/github/ReleaseBinariesTest.java]: looks like this is one-time variable

File: src/test/java/com/rultor/agents/github/ReleaseBinariesTest.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+     * @return Talk
+     * @throws IOException If fails
+     */
+    private static Talk talk(final Issue issue, final String tag,
+        final File dir)
+        throws IOException {
+        final Talk talk = new Talk.InFile();
+        final String identifier = "id";
+        talk.modify(
+            new Directives().xpath("/talk")
```
Comment (longtimeago) [inline src/test/java/com/rultor/agents/github/ReleaseBinariesTest.java]: Are you sure all following nodes and attributes are relevant to the test?
I'm not sure about these:
```
.attr("later", Boolean.TRUE.toString())
.add("daemon").attr(identifier, "123")
.add("title").set("merge").up()
.add("script").set("empty").up()
.add("code").set("-7").up()
.add("href").set("http://test2").up()
.add("github-issue").set(Integer.toString(issue.number())).up()
```
These directives hide what is really matter in the test. As far as I understand, the test should work without all of them.
Correct me if I wrong

**Comment by longtimeago** [comment]
@krzyk Please, see 2 remarks in the code above

File: src/test/java/com/rultor/agents/github/ReleaseBinariesTest.java (hunk 2 above)
Comment (krzyk) [inline src/test/java/com/rultor/agents/github/ReleaseBinariesTest.java]: - `.add("daemon").attr(identifier, "123")` is required to have the `dir` and xsd forces me to add `id` attribute
- `.add("title").set("merge").up()` required by xsd
- `.add("script").set("empty").up()` required by xsd
- `.add("href").set("http://test2").up()` required by xsd
- `.add("github-issue").set(Integer.toString(issue.number())).up()` required by xsd

Comment (longtimeago) [inline src/test/java/com/rultor/agents/github/ReleaseBinariesTest.java]: ahr ... forgot about xsd ...

**Comment by krzyk** [comment]
@longtimeago here's an update

**Comment by longtimeago** [comment]
@rultor please, merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by longtimeago** [comment]
@yegor256 shouldn't the PR be paid? 14 days passed

**Comment by yegor256** [comment]
@longtimeago looks like it was merged just yesterday... 14 days since what? :)

**Comment by longtimeago** [comment]
@yegor256 ups ... wrong issue, sorry

**Comment by alex-palevsky** [comment]
@longtimeago thanks, I just added **16 mins** to your account, payment `51586794`; review comments (c=1) added as a bonus; +16 added to your rating, at the moment it is: [+1690](http://www.netbout.com/b/35024?open=rating)

**Comment by alex-palevsky** [comment]
@rultor deploy pls

**Pull request merged**